pkg/printutil/printutil.go

```diff
@@ -136,3 +170,40 @@ func strSliceToInterSlice(ss []string) []interface{} {
 	}
 	return is
 }
+
+func (t *Table) createHeaderPadding() {
+	if len(t.headerPadding) == 0 {
+		for _, num := range t.Header {
+			colLength := len(num) + 5
+			t.headerPadding = append(t.headerPadding, colLength)
+		}
+	}
+}
+
+func (t *Table) coalesceHeaderAndRowPadding(val int, i int) {
+	if val < t.headerPadding[i] {
+		t.altPadding[i] = t.headerPadding[i]
+	} else {
+		t.headerPadding[i] = t.altPadding[i]
+	}
+}
+
+func (t *Table) dynamicPadding(rows []TempRow) {
```

Comment:
if you make []TempRow an attribute on the Table struct you can avoid having to pass an arg in here